bindings/go/plugin/go.mod

```diff
@@ -10,15 +10,19 @@ require (
 	ocm.software/open-component-model/bindings/go/descriptor/runtime v0.0.0-20250521120014-424308d81e7b
 	ocm.software/open-component-model/bindings/go/descriptor/v2 v2.0.0-20250520105023-82a459e39e46
 	ocm.software/open-component-model/bindings/go/runtime v0.0.1
+	ocm.software/open-component-model/cli v0.0.0-20250521134817-41f55c131044
```

Comment:
this is definetly incorrect, CLI should not be a dependency to plugin

Reply:
https://github.com/open-component-model/open-component-model/pull/171